fixtures/heredoc_ending_with_curly_expected.rb

```diff
@@ -1,4 +1,4 @@
-raise(<<-EOM.gsub(/^\s+\|/, ""))
+raise <<-EOM.gsub(/^\s+\|/, "")
```

Comment:
This changes because the current logic is “never use parens with `raise`”. For `super` and `yield` we currently follow whatever the original source did, and we could also do that for `raise` if we didn't want to change this fixture.